cirq/sim/density_matrix_simulator_test.py

```diff
@@ -656,3 +656,17 @@ def test_works_on_pauli_string():
     np.testing.assert_allclose(result.reshape(4, 4),
                                np.diag([0, 0, 0, 1]),
                                atol=1e-8)
+
+
+def test_str():
+    q0 = cirq.LineQubit(0)
+    final_simulator_state = cirq.DensityMatrixSimulatorState(
+        density_matrix=np.ones((2, 2)) * 0.5, qubit_map={q0: 0})
+    result = cirq.DensityMatrixTrialResult(
+        params=cirq.ParamResolver({}),
+        measurements={},
+        final_simulator_state=final_simulator_state)
+
+    assert str(result).startswith('measurements: (no measurements)\n'
```

Comment:
Why does this call `starts_with`?  Use `==` instead.

Reply:
Done.